### DISCUSSION TITLE
blog/%EC%B2%AB%EA%B8%80-%EA%B9%83%ED%97%88%EB%B8%8C-astro-%EB%B8%94%EB%A1%9C%EA%B7%B8%EB%A5%BC-%EC%97%B4%EC%97%88%EC%8A%B5%EB%8B%88%EB%8B%A4

### DISCUSSION THREAD

**Discussion by giscus** [YieumYoon/YieumYoon.github.io#1] [Announcements]
# blog/%EC%B2%AB%EA%B8%80-%EA%B9%83%ED%97%88%EB%B8%8C-astro-%EB%B8%94%EB%A1%9C%EA%B7%B8%EB%A5%BC-%EC%97%B4%EC%97%88%EC%8A%B5%EB%8B%88%EB%8B%A4

Astro 깃허브 블로그 시작, Github를 선택한 이유, Astro를 프레임워크로 선택한 이유

https://4321-idx-yieumyoon-github-io-1717681324437.cluster-qpa6grkipzc64wfjrbr3hsdma2.cloudworkstations.dev/blog/%EC%B2%AB%EA%B8%80-%EA%B9%83%ED%97%88%EB%B8%8C-astro-%EB%B8%94%EB%A1%9C%EA%B7%B8%EB%A5%BC-%EC%97%B4%EC%97%88%EC%8A%B5%EB%8B%88%EB%8B%A4

**Comment by YieumYoon** [comment]
test comment
